extensions/wikia/Recirculation/js/helpers/LiftigniterHelper.js

```diff
@@ -56,6 +56,7 @@ define('ext.wikia.recirculation.helpers.liftigniter', [
 					}
 
 					item.meta = options.widget;
+					// item.sponsor = "Hulu";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)